bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/Ngs2Strategy.scala

```diff
@@ -162,12 +166,28 @@ final case class Ngs2Strategy(
         // that is repositioned at each science offset and then the intersection
         // of these will be the valid range.  For now though, we'll just use the
         // candidates within the (0, 1') range from GemsCandidates.
```

Comment:
We'll need to update this comment a bit.  Maybe:

> REL-3745: The code below does its own range and magnitude filtering of CWFS candidates instead of just calling `GemsCandidates.groupAndValidate`.  We make a new special-purpose validator using a PatrolField.fromRadiusLimits(0, 1.3') so that candidates slightly outside of range will be shown alongside strictly valid candidates.  Regardless, the candidate pool is ultimately filtered down to valid targets for selection.